scripts/next-changelogs.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-await-in-loop */
+const { execSync } = require("child_process");
+
+const getLatestReleaseTags = () => {
+  const tags = execSync("git tag --sort=-creatordate", { encoding: "utf8" });
+  return tags
+    .split("\n")
+    .map((t) => t.trim())
+    .filter(
+      (tag) =>
+        tag.includes("-next.") ||
+        tag.match(/^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$/),
+    );
+};
+
+class NextChangelogsPlugin {
+  name = "next-changelogs";
+
+  apply(auto) {
+    auto.hooks.next.tapPromise(this.name, async ({ dryRun }) => {
+      const [latest, second] = getLatestReleaseTags();
```

Comment:
Can any edge case cause `getLatestReleaseTags` to return less than two tags? If so, this will fail when destructuring..

Reply:
In theory yes but since this repo already has multiple releases we should always have a to <-> from